gnovm/pkg/gnomod/file_test.go

```diff
@@ -14,7 +14,7 @@ import (
 	"golang.org/x/mod/module"
 )
 
-const testRemote string = "test3.gno.land:36657"
+const testRemote string = "test3.gno.land:26657"
```

Comment:
Is this changed on test3?

Reply:
If PR is approved and ready to be merged, i update it and it will works